src/ctia/entity/event.clj

```diff
@@ -62,22 +69,89 @@
 
 (def EventGetParams EventFieldsParam)
 
+(defn same-bucket? [bucket event]
+  (let [max-seconds (get-in @properties [:ctia :events :timeline :max-seconds] 5)
+        from        (t/minus (:from bucket) (t/seconds max-seconds))
+        to          (t/plus (:to bucket) (t/seconds max-seconds))]
```

Comment:
Perhaps it would be safer to use `(:from bucket)` here too. Otherwise we could end with no time limit bucket. See further discussion about event at times t1 ... tn each occurring at 2/3 the time of timeline max-seconds.